ci-operator/config/tnozicka/openshift-acme/master.json

```diff
@@ -0,0 +1,77 @@
+{
+  "base_images": {
+    "os": {
+      "cluster": "https://api.ci.openshift.org",
+      "namespace": "openshift",
+      "name": "centos",
+      "tag": "7"
+    }
+  },
+  "test_base_image": {
+    "cluster": "https://api.ci.openshift.org",
+    "namespace": "openshift",
+    "name": "release",
+    "tag": "golang-1.10"
+  },
+
+  "canonical_go_repository": "github.com/tnozicka/openshift-acme",
+
+  "binary_build_commands": "make build GOFLAGS='-v'",
+  "test_binary_build_commands": "make build GOFLAGS='-v -race'",
+
+  "images": [
+    {
+      "from": "os",
+      "to": "openshift-acme",
+      "dockerfile_path": "images/Dockerfile",
+      "inputs": {
+        "bin": {
+          "as": ["bin"]
```

Comment:
You're not actually using anything from the `bin` image here -- what is this build doing?

Reply:
Form what I've read in the operator source code this is suppose to use that image to replace a step in multi stage build here https://github.com/tnozicka/openshift-acme/commit/11fba481be09b7ada4aab2771054d2a3e3c7c68e#diff-475e714b0c3678141c59dab9961e8841R1

Reply:
Oh right, you use the multi-stage. Disregard